modules/compliance-priorityclass.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-advanced.adoc
+
+:_content-type: PROCEDURE
+[id="compliance-priorityclass_{context}"]
+= Setting `PriorityClass` for `ScanSetting` scans
+
+In large scale environments, the default `PriorityClass` can be too low to guarantee Pods execute scans on time. The Compliance Operator requires a high priority to maintain compliance in a cluster.
```

Comment:
nit: I'm probably splitting hairs here. Technically, the compliance operator doesn't need `PriorityClass` to function properly. Although, it would make sense for users who value compliance to set the `PriorityClass` for the operator, it's not expected.

I wouldn't want someone to interpret this as a technical dependency, as opposed to something that just makes sense for environments who place a high priority on compliance.

Since we're already leading the sentence with a situation, we could expand on it:

```
In large scale environments, the default `PriorityClass` can
be too low to guarantee Pods execute scans on time. For
clusters that must maintain compliance or guarantee
automated scanning, we recommend setting the `PriorityClass`
to ensure the Compliance Operator is always given priority
in resource constrained situations.
```

Again, I'm sure that could be worded better, but trying to make it clear the `PriorityClass` bit isn't a functional requirement of the Compliance Operator implementation, but more so a tool for deployments who consider security and compliance a requirement (e.g., I'll lose $X if my scans aren't automated, or if my compliance falls below a certain threshold.)